### DISCUSSION TITLE
Идея для плейлистов

### DISCUSSION THREAD

**Discussion by pavelglazunov** [MarshalX/yandex-music-api#523] [Идеи]
На данный момент, в мобильном приложении Яндекс музыки существует 2 варианта прослушивания:
1) в порядке плейлиста
2) в случайном порядке
Но очень часто бывает желание послушать свою музыку, но в полностью случайном порядке. Во втором варианте, при перемешивании плейлиста, он зацикливается
ИДЕЯ: Сделать дополнительный режим, в котором все треки будут в случайном порядке, но при достижении конца плейлиста, они будут еще раз перемешиваться. Таким образом получиться слушать свою любимую музыку, в полностью случайным порядке

**Comment by darkkeks** [comment]
Поддерживаю
@MarshalX, когда ждать такую фичу в мобильном приложении Яндекс.Музыки?

**Reply by darkkeks** [reply]
Было бы замечательно, если бы такая фича еще появилась в Spotify
@MarshalX сделаешь?
